### ISSUE TITLE
TransformerTorchEncoder index error 

### ISSUE THREAD

**Issue by Kavan72** [jina-ai/jina-hub#1483]
encode.yml:-
```
!TransformerTorchEncoder
with:
  pooling_strategy: cls
  pretrained_model_name_or_path: distilbert-base-cased
metas:
  on_gpu: true
```
![image](https://user-images.githubusercontent.com/19048640/103218507-2e18a100-4941-11eb-9bde-306b6dd99587.png)

**Comment by JoanFM** [comment]
Hey @Kavan72,

can u check if u have these versions?

transformers==4.0.1
torch==1.7.1

**Comment by Kavan72** [comment]
even after installing same version still not working

**Comment by JoanFM** [comment]
can u share a minimum example with the list of versions (jina, transformers, etc...) so that we can reproduce?

**Comment by Kavan72** [comment]
https://github.com/Kavan72/jina_test check this

**Comment by JoanFM** [comment]
Hey @Kavan72 ,

I have been able to reproduce the issue.

There is a problem with dimensions when `batch_size` is 1. As a workaround to serve your purpose, I recommend that you can increase the `batch_size` > 1

**Comment by Kavan72** [comment]
@JoanFM thank for you time and it's working with `>1` batch_size

**Comment by JoanFM** [comment]
Hey @Kavan72 , #1501 should also fix for batch_size 1

**Comment by JoanFM** [comment]
Closed by #1501

**Comment by Kavan72** [comment]
thank you
